packages/serverless-components/domain/serverless.js

```diff
@@ -191,15 +197,17 @@ class Domain extends Component {
         );
         await removeDomainFromCloudFrontDistribution(clients.cf, domainState);
 
-        this.context.debug(
-          `Removing CloudFront DNS records for domain ${domainState.domain}`
-        );
-        await removeCloudFrontDomainDnsRecords(
-          clients.route53,
-          domainState.domain,
-          domainHostedZoneId,
-          domainState.url.replace("https://", "")
-        );
+        if (domainHostedZoneId) {
+          this.context.debug(
+            `Removing CloudFront DNS records for domain ${domainState.domain}`
+          );
+          await removeCloudFrontDomainDnsRecords(
+            clients.route53,
+            domainState.domain,
+            domainHostedZoneId,
+            domainState.url.replace("https://", "")
+          );
+        }
```

Comment:
Not sure if it's useful to have a message in the `else` case too?

Reply:
User by now knows there is no DNS record to be removed but i'll add one